src/style.css

```diff
@@ -0,0 +1,149 @@
+:root {
+  font-family: "Roboto", sans-serif;
+  --dark-blue: #060b35;
+  --lghit-grey: #616070;
+  --lghit-blue: #00acdc;
+  --white: #fff;
+}
+
+body {
+  margin: 0;
+  display: flex;
+}
+
+.card,
+.card--hover {
+  margin: 40px;
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+  background-color: #fff;
```

Comment:
redundant

Reply:
Remove this line, cause the default background color is white, and you don't need to specify it.